ci-operator/populate-secrets-from-bitwarden.sh

```diff
@@ -130,206 +130,22 @@ function field_value() {
-	"$( format_attachment "packet.net" client.crt matchbox-client.crt )" \
-	"$( format_attachment "packet.net" client.key matchbox-client.key )"
-
 target_cloud="azure"
 update_secret generic "cluster-secrets-${target_cloud}"                                 \
 	"$( format_attachment "quay.io" pull-secret )"                                      \
 	"$( format_attachment "os4-installer.openshift-ci.azure" osServicePrincipal.json )" \
 	"$( format_attachment "jenkins-ci-iam" ssh-privatekey )"                            \
 	"$( format_attachment "jenkins-ci-iam" ssh-publickey )"
 
-# DSNs for tools reporting failures to Sentry
-update_secret generic "sentry-dsn" "$( format_field_value "sentry" "ci-operator" "ci-operator" )"
-
-# codecov.io tokens we store for teams
-update_secret generic "redhat-developer-service-binding-operator-codecov-token" "$( format_field_value "codecov-tokens" redhat-developer-service-binding-operator token )"
-
-# collects all the secrets for build farm
-update_secret generic "build-farm-credentials" \
-	"$( format_field_value build_farm_01_cluster "github_client_secret" "build01_github_client_secret" )" \
-	"$( format_attachment "build_farm" build01_ci_reg_auth_value.txt )" \
-	"$( format_attachment "build_farm" build01_build01_reg_auth_value.txt )" \
-	"$( format_attachment "build_farm" sa.deck.build01.config )" \
-	"$( format_attachment "build_farm" sa.hook.build01.config )" \
-	"$( format_attachment "build_farm" sa.plank.build01.config )" \
-	"$( format_attachment "build_farm" sa.sinker.build01.config )" \
-	"$( format_attachment "build_farm" sa.ca-cert-issuer.build01.config )" \
-	"$( format_attachment "build_farm" sa.config-updater.build01.config )"
-
-# collects all the secrets for ci-operator
-update_secret generic "apici-ci-operator-credentials" \
-	"$( format_attachment "build_farm" sa.ci-operator.apici.config )"
-
-# collects all the secrets for boskos
-update_secret generic "boskos-credentials" \
-	"$( format_field_value "boskos-oauth-proxy" "boskos-username" "username" )" \
-	"$( format_field_value "boskos-oauth-proxy" "boskos-password" "password" )" \
-	"$( format_attachment "boskos-oauth-proxy" ci.htpasswd )"
-
-# collects all the secrets for aws iamuser openshift-ci-robot iamuser under account ci-infra
-# it is used for issue CA certificates for build01
-update_secret generic "aws-ci-infra-openshift-ci-robot-credentials" \
-	"$( format_field_value "aws_ci_infra_openshift-ci-robot" "AWS_ACCESS_KEY_ID" "AWS_ACCESS_KEY_ID" )" \
-	"$( format_field_value "aws_ci_infra_openshift-ci-robot" "AWS_SECRET_ACCESS_KEY" "AWS_SECRET_ACCESS_KEY" )"
-
 # Configuration for the .git-credentials used by the release controller to clone
 # private repositories to generate changelogs
 oc -n "ci-release" create secret generic "git-credentials" "--from-literal=.git-credentials=https://openshift-bot:$( field_value "openshift-bot" "GitHub OAuth Token" "oauth" )@github.com" --dry-run -o yaml | oc apply -f -
```

Comment:
@stevekuznetsov 

Maybe create new fields with the result of string manipulation?

Reply:
Agreed.